static/src/javascripts/lib/detect.js

```diff
@@ -219,6 +219,15 @@ const hasCrossedBreakpoint = (includeTweakpoint: boolean): Function => {
     };
 };
 
+// Will return true for any Safari browser (including Mobile Safari)
+// For an explanation of this solution, see https://stackoverflow.com/questions/7944460/detect-safari-browser/42189492
+const isSafari =
```

Comment:
Hey, I wonder whether feature detection might be better here? Using `window.ApplePaySession`.
https://developer.apple.com/documentation/apple_pay_on_the_web/apple_pay_js_api/checking_for_apple_pay_availability

Reply:
Great suggestion!

Reply:
Done!

I've decided to just check for the availablity of the Apple Pay API, rather than whether they have it set up. This is for 2 reasons:

- The function you can call on the `window.ApplePaySession` object to determine if they have Apple Pay set up is async, meaning that we would have to call this on the first page load and store something in the session storage, which wouldn't be guaranteed to happen in time for the initial epic display
- If someones got a device that Apple Pay can be used on but hasn't set it up yet, it can't hurt to let them know that we take it, they may get it in the future and it may even encourage them to set it up

Reply:
@ademuk what do you think? If you are fine with this, coud you +1 it?

Reply:
Yes, that makes sense regarding detecting available vs. setup.

Other than minor comment above. LGTM 👍

Reply:
thanks a lot for weighing in here @ademuk - your suggestion is definitely the right way to do it!